mappings/net/minecraft/block/Material.mapping

```diff
@@ -4,45 +4,45 @@ CLASS net/minecraft/class_3614 net/minecraft/block/Material
 	FIELD field_15913 COBWEB Lnet/minecraft/class_3614;
 	FIELD field_15914 STONE Lnet/minecraft/class_3614;
 	FIELD field_15915 BUBBLE_COLUMN Lnet/minecraft/class_3614;
-	FIELD field_15916 SAND Lnet/minecraft/class_3614;
+	FIELD field_15916 AGGREGATE Lnet/minecraft/class_3614;
 	FIELD field_15917 SPONGE Lnet/minecraft/class_3614;
 	FIELD field_15918 REDSTONE_LAMP Lnet/minecraft/class_3614;
 	FIELD field_15919 PORTAL Lnet/minecraft/class_3614;
 	FIELD field_15920 WATER Lnet/minecraft/class_3614;
 	FIELD field_15921 UNUSED_PLANT Lnet/minecraft/class_3614;
 	FIELD field_15922 LAVA Lnet/minecraft/class_3614;
 	FIELD field_15923 LEAVES Lnet/minecraft/class_3614;
-	FIELD field_15924 PART Lnet/minecraft/class_3614;
+	FIELD field_15924 DECORATION Lnet/minecraft/class_3614;
 	FIELD field_15925 CACTUS Lnet/minecraft/class_3614;
-	FIELD field_15926 SEAGRASS Lnet/minecraft/class_3614;
+	FIELD field_15926 REPLACEABLE_AQUATIC_PLANT Lnet/minecraft/class_3614;
 	FIELD field_15927 STRUCTURE_VOID Lnet/minecraft/class_3614;
-	FIELD field_15928 PACKED_ICE Lnet/minecraft/class_3614;
+	FIELD field_15928 DENSE_ICE Lnet/minecraft/class_3614;
 	FIELD field_15929 liquid Z
 	FIELD field_15930 EGG Lnet/minecraft/class_3614;
 	FIELD field_15931 WOOL Lnet/minecraft/class_3614;
 	FIELD field_15932 WOOD Lnet/minecraft/class_3614;
 	FIELD field_15933 PISTON Lnet/minecraft/class_3614;
 	FIELD field_15934 SNOW_BLOCK Lnet/minecraft/class_3614;
 	FIELD field_15935 PLANT Lnet/minecraft/class_3614;
-	FIELD field_15936 CLAY Lnet/minecraft/class_3614;
+	FIELD field_15936 MOB_DERIVED Lnet/minecraft/class_3614;
 	FIELD field_15937 CAKE Lnet/minecraft/class_3614;
-	FIELD field_15938 BAMBOO_SAPLING Lnet/minecraft/class_3614;
+	FIELD field_15938 BAMBOO_SHOOT Lnet/minecraft/class_3614;
 	FIELD field_15939 blocksLight Z
 	FIELD field_15940 blocksMovement Z
-	FIELD field_15941 EARTH Lnet/minecraft/class_3614;
+	FIELD field_15941 TOPSOIL Lnet/minecraft/class_3614;
```

Comment:
This is used by dirt as well. I dont think this is correct

Reply:
Dirt is considered topsoil as well

Reply:
Dirt is not _exclusively_ topsoil.

Reply:
yes, but everything with this material can be described as topsoil. The purpose of this name is to narrow down the overgeneralized name of "earth" to make it more descriptive. "Earth" is far too broad, so if you have an alternative name I'm open to suggestions.

Reply:
Not really an improvement

Reply:
"Earth" is too broad of a name. "Topsoil" is closer to what the blocks are.

Reply:
Shouldn't "topsoil" be two words? `TOP_SOIL`

Reply:
Topsoil is one word.

Reply:
Still don't think this is an improvement.

Reply:
Changed to SOIL as per feedback